rcm_nexus/command.py

```diff
@@ -2,19 +2,22 @@
 from rcm_nexus.session import Session
 import rcm_nexus.config as config
 import rcm_nexus.repo as repos
-import rcm_nexus.group as groups
 import rcm_nexus.archive as archive
 import rcm_nexus.staging as staging
 import os.path
 import sys
-import re
 import click
+import shutil
 import tempfile
 
 RELEASE_GROUP_NAME = 'product-ga'
 TECHPREVIEW_GROUP_NAME = 'product-techpreview'
 PRERELEASE_GROUP_NAME = 'product-earlyaccess'
 
+EARLY_ACCESS_PROFILE_ID = "537461ce29c60"
```

Comment:
Would these be better suited to the configuration file?

Reply:
Probably yes. As far as I understand it, the values are not expected to change and are product independent. But you're right that config file is probably a better place for it.